examples/src/app/code-editor.mjs

```diff
@@ -148,6 +172,8 @@ class CodeEditor extends TypedComponent {
             // keybindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.Enter],
             // contextMenuGroupId: 'navigation',
             contextMenuOrder: 1.5,
+            // @ts-ignore
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Wait...why are we running TypeScript ESLint rules on JS code? The examples browser is no longer written in TS, thanks to @kungfooman